framework/packages/abstract-std/src/core/manager.rs

```diff
@@ -174,7 +138,6 @@ impl ModuleInstallConfig {
 }
 
 /// Manager execute messages
-#[cw_ownable::cw_ownable_execute]
```

Comment:
Would you prefer to keep it as macro or as variant? I personally dislike macros that introduce message variants, as they are harder to read IMO

Reply:
Agreed! let's get rid of the macro